diff -up v8-7.4.284/BUILD.gn.0000~ v8-7.4.284/BUILD.gn
--- v8-7.4.284/BUILD.gn.0000~	2019-03-05 16:06:08.000000000 +0100
+++ v8-7.4.284/BUILD.gn	2019-03-05 16:24:17.981154681 +0100
@@ -3283,6 +3283,7 @@ v8_component("v8_libbase") {
 
   if (is_component_build) {
     defines = [ "BUILDING_V8_BASE_SHARED" ]
+    output_extension = "so.7"
   }
 
   if (is_posix || is_fuchsia) {
@@ -3415,6 +3416,7 @@ v8_component("v8_libplatform") {
 
   if (is_component_build) {
     defines = [ "BUILDING_V8_PLATFORM_SHARED" ]
+    output_extension = "so.7"
   }
 
   public_configs = [ ":libplatform_config" ]
@@ -3681,6 +3683,8 @@ if (is_component_build) {
       "src/v8dll-main.cc",
     ]
 
+    output_extension = "so.7"
+
     public_deps = [
       ":v8_base",
       ":v8_maybe_snapshot",
@@ -3698,6 +3702,8 @@ if (is_component_build) {
       "src/v8dll-main.cc",
     ]
 
+    output_extension = "so.7"
+
     public_deps = [
       ":torque_base",
       ":torque_ls_base",
