diff -up v8-8.4.154/BUILD.gn.2~ v8-8.4.154/BUILD.gn
--- v8-8.4.154/BUILD.gn.2~	2020-04-24 16:18:31.786032144 +0200
+++ v8-8.4.154/BUILD.gn	2020-04-24 16:26:55.808244463 +0200
@@ -1274,7 +1274,6 @@ if (v8_verify_torque_generation_invarian
 
 group("v8_maybe_icu") {
   if (v8_enable_i18n_support) {
-    public_deps = [ "//third_party/icu" ]
   }
 }
 
@@ -4253,9 +4254,12 @@ if (v8_enable_i18n_support) {
       deps = [
         ":v8_libbase",
         "//build/win:default_exe_manifest",
-        "//third_party/icu",
       ]
 
+      libs = [ "icuuc", "icui18n" ]
+
+      defines = [ "USING_SYSTEM_ICU=1" ]
+
       configs = [ ":internal_config" ]
     }
   }
diff -up v8-8.4.154/test/cctest/BUILD.gn.2~ v8-8.4.154/test/cctest/BUILD.gn
--- v8-8.4.154/test/cctest/BUILD.gn.2~	2020-04-24 16:18:31.786032144 +0200
+++ v8-8.4.154/test/cctest/BUILD.gn	2020-04-24 16:27:28.288664191 +0200
@@ -413,7 +413,8 @@ v8_source_set("cctest_sources") {
 
   if (v8_enable_i18n_support) {
     defines += [ "V8_INTL_SUPPORT" ]
-    public_deps += [ "//third_party/icu" ]
+    libs = [ "icuuc", "icui18n" ]
+    defines += [ "USING_SYSTEM_ICU=1" ]
   } else {
     sources -= [ "test-intl.cc" ]
   }
diff -up v8-8.4.154/tools/debug_helper/BUILD.gn.omv~ v8-8.4.154/tools/debug_helper/BUILD.gn
--- v8-8.4.154/tools/debug_helper/BUILD.gn.omv~	2020-04-24 16:40:39.888209875 +0200
+++ v8-8.4.154/tools/debug_helper/BUILD.gn	2020-04-24 16:41:07.189579862 +0200
@@ -92,7 +92,8 @@ v8_component("v8_debug_helper") {
 
   configs = [ ":internal_config" ]
   if (v8_enable_i18n_support) {
-    configs += [ "//third_party/icu:icu_config" ]
+    defines = [ "USING_SYSTEM_ICU=1" ]
+    libs = [ "icuuc", "icui18n" ]
   }
 
   remove_configs = [ "//build/config/compiler:no_rtti" ]
